CenterForOpenScience/osf.io
# [Feature] Update archiver wbv1 [OSF-7284]

## Purpose
Archiver needs to use WBV1 instead of WBV0


## Changes
Swaps the routes and makes it work.


## Side effects
Minor refactor to simplify and clean up.



## Ticket


https://openscience.atlassian.net/browse/OSF-7284

## QA Notes
Make a registrations and ensure that files in various addons are copied over properly. Registering a project with all of the storage addons set up should be sufficient.
